server/follow/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict'
+
+const serie = require('fastseries')()
+
+const {
+  follow: followSchema,
+  unfollow: unfollowSchema,
+  followers: followersSchema
+} = require('./schemas')
+const FollowService = require('./FollowService')
+
+module.exports = function (fastify, opts, next) {
+  // See user/index.js for some little explainations
+  serie(
+    fastify,
+    [
+      registerEnv,
+      registerRedis,
+      decorateWithTweetService,
+      decorateWithUserClient,
+      registerRoutes
+    ],
+    opts,
+    next
+  )
+}
```

Comment:
can you refactor all of this to not use series? You can just issues a bunch of "register" calls.

Reply:
This cannot be done:
`registerRedis` depends on `registerEnv`  and the decorators depend on the two registration.